honeybadgermpc/reed_solomon_wb.py

```diff
@@ -213,7 +213,7 @@ def no_solution(a):
 # and return the row index of the 1 if it exists
 def is_pivot_column(a, j):
     i = 0
-    while a[i][j] == 0 and i < len(a):
+    while i < len(a) and a[i][j] == 0:
```

Comment:
That's awesome! The fix was way more straightforward than I thought it would be.